samples: nrf9160: Increase MQTT keepalive timeout (#3400)
nrfconnect/sdk-nrf: samples/nrf9160/aws_iot/README.rst

Configures the time interval between each publishing of the message.

> .. warning::
> 
>   The sample sets the option :option:`CONFIG_MQTT_KEEPALIVE` to the maximum allowed value specified by AWS IoT Core, 1200 seconds (20 minutes).
>   This is to limit the IP traffic between the device and the AWS IoT message broker in order to encourage a low power sample.
>   However, it should be noted that in certain LTE networks, the NAT timeout can be conciderable lower than 1200 seconds. So as a recommendation, and to prevent the likelyhood of
>   getting thrown out of the connection "unexpectedly", option :option:`CONFIG_MQTT_KEEPALIVE` should be set to the lowest value of the aforementioned timeout limits. (Maximum allowed MQTT keepalive and NAT timeout).

Reviewer: ```suggestion
.. note::

   The sample sets the option :option:`CONFIG_MQTT_KEEPALIVE` to the maximum allowed value, 1200 seconds (20 minutes) as specified by AWS IoT Core.
  This is to limit the IP traffic between the device and the AWS IoT message broker for supporting a low power sample.
   However, note that in certain LTE networks, the NAT timeout can be considerably lower than 1200 seconds.
   So as a recommendation, and to prevent the likelihood of getting disconnected unexpectedly, the option :option:`CONFIG_MQTT_KEEPALIVE` must be set to the lowest of the aforementioned timeout limits (Maximum allowed MQTT keepalive and NAT timeout).
```
